waffle/models.py

```diff
@@ -13,6 +14,9 @@
 from waffle import managers
 from waffle.utils import get_setting, keyfmt, get_cache
 
+LOGGER_NAME = 'waffle'
+logger = logging.getLogger(LOGGER_NAME)
```

Comment:
Can this be a little simpler, just `logger = logging.getLogger('waffle')`? We're not re-using `LOGGER_NAME` anywhere and contextually it's obvious that it's the logger name.